File: Silver/Cooper/Android/FirstAndroidApp/DisplayMessageActivity.swift
import java.util
import android.app
import android.content
import android.os
import android.util
import android.view
import android.widget

public class DisplayMessageActivity : Activity {
		
	public override func onCreate(_ savedInstanceState: Bundle!) {
		
		super.onCreate(savedInstanceState);
		
		let intent = getIntent();
		let message = intent.getStringExtra(MainActivity.EXTRA_MESSAGE);
		
		let textView = TextView(self);
		textView.TextSize = 40;
		textView.Text = message;
		ContentView = textView;
	}
}

File: Iodine/Cooper/Android/FirstAndroidApp/DisplayMessageActivity.java
package org.me.firstandroidapp;

import java.util.*;
import android.app.*;
import android.content.*;
import android.os.*;
import android.util.*;
import android.view.*;
import android.widget.*;

public class DisplayMessageActivity extends Activity
{
	@Override
	public void onCreate(Bundle savedInstanceState)
	{
		super.onCreate(savedInstanceState);

		Intent intent = getIntent();
		String message = intent.getStringExtra(MainActivity.EXTRA_MESSAGE);

		TextView textView = new TextView(this);
		textView.TextSize = 40;
		textView.Text = message;
		ContentView = textView;
	}

}